src/pages/LendingStats/LendingStatsPage.vue

```diff
@@ -82,6 +85,7 @@ export default {
 	},
 	data() {
 		return {
+			headerTheme: lightHeader,
```

Comment:
Please revert this page. This is legacy stack and all of it's siblings in Portfolio have a "Green" header.